packages/frontity-org-theme/src/components/background-with-triangles.tsx

```diff
@@ -0,0 +1,118 @@
+import { connect, css, styled } from "frontity";
+import { Connect } from "frontity/types";
+import React from "react";
+
+import FrontityOrg from "../../types";
+
+const TriangleComp: React.FC<{
+  top?: string;
+  position: string;
+  topTriangleOpacity?: string;
+}> = ({ position, top, topTriangleOpacity }) => (
+  <Triangle
+    className={`position-${position}`}
+    top={top}
+    topTriangleOpacity={topTriangleOpacity}
+  >
+    <Triangle className="inner" />
+  </Triangle>
+);
+
+const BackgroundWithTriangles: React.FC<Connect<
+  FrontityOrg,
+  { height?: string }
+>> = ({ state, height }) => {
+  const data = state.source.get(state.router.link);
+  const page = state.source.page[data.id];
+
+  const { position, top, top_triangle_opacity: topTriangleOpacity } = page?.acf[
+    "background_triangles"
+  ];
+
+  return (
+    <Container height={height}>
+      {position !== "both-sides" && position !== "none" ? (
+        <TriangleComp
+          topTriangleOpacity={topTriangleOpacity}
+          position={position}
+          top={top}
+        />
+      ) : null}
+
+      {position !== "both-sides" &&
```

Comment:
I think this logic is doing just the opposite, it should be `position === "both-sides"` right?

Reply:
Oh yeah! I changed that to debug something. I'll fix it.